extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -166,13 +226,15 @@ export async function importArchiveDatabase(
  * or in the local filesystem.
  *
  * @param databaseUrl URL from which to grab the database
+ * @param requestHeaders Headers to send with the request
  * @param databaseManager the DatabaseManager
  * @param storagePath where to store the unzipped database.
  * @param progress callback to send progress messages to
  * @param token cancellation token
  */
 async function databaseArchiveFetcher(
   databaseUrl: string,
+  requestHeaders: { [key: string]: string },
```

Comment:
Clever, I see this lets you share the logic of fetching and unzipping archives between GitHub and LGTM.

Reply:
Yes! Props to @robertbrignull ⚡